src/Clan.ts

```diff
@@ -245,4 +298,73 @@ export class Clan {
     );
     return result.includes("You contributed");
   }
+
+  /**
+   * Take items from the stash
+   * 
+   * This function will also take equivalent foldables if the original item cannot be found
+   * 
+   * @param items Items to take
+   * @returns Items successfully taken
+   */
+  @validate
+  take(items: Item[]): Item[] {
+    return items.filter((item) => {
+      if (have(item)) return false;
+
+      const foldGroup = getFoldGroup(item);
+
+      if (foldGroup.some(fold => have(fold))) {
+        cliExecute(`fold ${item.name}`);
+        return false;
+      }
+
+      refreshStash();
+
+      return [item, ...foldGroup].some((matchingItem) => {
+        if (stashAmount(matchingItem) === 0) return false;
+        if (!takeStash(1, matchingItem)) return false;
```

Comment:
If `takeStash` fails for one item in a fold group, do we expect it to succeed for any others? If a fold group is 0-karma, the entire fold group is 0-karma. 

Similarly, is there value in drawing a distinction between items you already had and items you failed to yoink in `take()`?

Reply:
> If `takeStash` fails for one item in a fold group, do we expect it to succeed for any others? If a fold group is 0-karma, the entire fold group is 0-karma.
Probably not but I'm inclined to keep the code simple.

> Similarly, is there value in drawing a distinction between items you already had and items you failed to yoink in `take()`?
No I don't think so - the only thing we really care about is what we took so we can return it after